ortools/linear_solver/unittests/xpress_interface.cc

```diff
@@ -132,6 +142,23 @@ namespace operations_research {
       return (XPRSprob)solver_->underlying_solver();
     }
 
+    std::string getName(int n, int type) {
+      int namelength;
+      EXPECT_STATUS(XPRSgetintattrib(prob(), XPRS_NAMELENGTH, &namelength));
+
+      std::string name;
+      name.resize(8 * namelength + 1);
+      EXPECT_STATUS(XPRSgetnames(prob(), type, name.data(), n, n));
+
+      name.erase(std::find_if(name.rbegin(), name.rend(),
+                              [](unsigned char ch) {
+                                return !std::isspace(ch) && ch != '\0';
```

Comment:
https://en.cppreference.com/w/cpp/locale/isspace
[locale withespace](Checks if the given character is classified as a whitespace character by the given locale's [std::ctype](https://en.cppreference.com/w/cpp/locale/ctype) facet.)

Reply:
be specific